### ISSUE TITLE
Custom loss function involving TensorFlow integral 

### ISSUE THREAD

**Issue by FrancescoFrontino** [keras-team/keras#11505]
Hello :) 

I am trying to implement a custom loss function in Keras of this form:

[<a href="https://www.codecogs.com/eqnedit.php?latex=\int_{0}^{5000}&space;\left&space;(&space;x&space;-&space;y\_true&space;\right&space;)^2*f(x,&space;y\_pred)&space;dx" target="_blank"><img src="https://latex.codecogs.com/gif.latex?\int_{0}^{5000}&space;\left&space;(&space;x&space;-&space;y\_true&space;\right&space;)^2*f(x,&space;y\_pred)&space;dx" title="\int_{0}^{5000} \left ( x - y\_true \right )^2*f(x, y\_pred) dx" /></a>](url)

`
This is the snippet of code that I have write down, however It doesn't work:

    def custom_loss_function(y_true, y_pred):

      a = y_pred[..., 0]
      b = y_pred[..., 1]

      def f(y, x):
          return ((y_true-x)**2) * ( (b/(a)**b) * (x**(b-1)) * K.exp(-(x/a)**b) )

      x = K.constant([ 0., 5000. ])

      return K.mean(tf.contrib.integrate.odeint_fixed( f, 0., x, method = "rk4" ))

`

<img width="973" alt="schermata 2018-10-27 alle 17 16 12" src="https://user-images.githubusercontent.com/22856979/47605847-3f399080-da0c-11e8-81ab-1cb47458b8b5.png">

[error_custom_loss.pdf](https://github.com/keras-team/keras/files/2521916/error_custom_loss.pdf)

Addionally, I have uploaded a pdf showing all the error messages. 

Thank you so much.
